### ISSUE TITLE
Ignoring controller calibration

### ISSUE THREAD

**Issue by ferrybig** [bwiklund/gamepad.js#6]
Inside windows, you can configure controller calibration for controllers with unaligned axises, this website ignores this calibration, and reports a range of -0.9 to 0.2 with the middle of the range a -0.411765, what gives a wrong impression for controller support on the web

**Comment by My1** [comment]
I dont think it ignores it explicitly. I dunno which exact input API is used and what the browsers do in the end bit I think the problem starts there.
